test/end-to-end/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "end-to-end",
+  "version": "0.0.0",
+  "description": "weldr-web end-to-end automation test",
+  "scripts": {
+    "test": "mocha -R xunit -O output=result.xml"
+  },
+  "dependencies": {
```

Comment:
I don't understand the need for 2 package.json files (one here and another one in the top directory), can you explain? I get it that tests could be packaged on their own, maybe even they can live into a separate repository. 

If this is the case why do you add NPM dependencies to the top-level package.json file as well? This is what confuses me.